bin/bootstrap

```diff
@@ -86,7 +87,8 @@ if [[ $? != 0 ]]; then
   echo "Github SSH key not found. Generating..."
   echo -en "\n\033[1mEnter your the email address you'll use for Github:\033[0m "
   read email
-  ssh-keygen -f ~/.ssh/github -t rsa -b 4096 -C $email
+  # use recommended ed25519 over rsa
+  ssh-keygen -f ~/.ssh/github -t ed25519 -C $email
```

Comment:
This only runs for folks who have not setup github ssh yet, should we also add a path for folks to upgrade?

Reply:
@emmoistner, updated file name to force ed25519 key creation.